### PR TITLE
Adding Technical Documentation Versioning best practices

### PR DESCRIPTION
**Pull request by nate-double-u**
Adapting my [Technical Documentation Versioning slide show](https://technical-documentation-versioning.netlify.app/),  [nate-double-u/technical-documentation-versioning](https://github.com/nate-double-u/technical-documentation-versioning) as per @celestehorgan's suggestion.

Deploy preview: https://github.com/cncf/techdocs/blob/23-versioning/howto/versioning-documentation.md

Contributes to #23

### CODE REVIEW

File: howto/versioning-documentation.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+---
+title: Technical Documentation Versioning with Hugo & Netlify
+author: [Nate Waddington](https://github.com/nate-double-u)
+date: 2021-01-30
+update: 2021-07-05
+license: CC-BY-4.0
+---
+
```
Comment (nate-double-u) [inline howto/versioning-documentation.md:8]: This is an artifact of the slideshow markdown file. We can remove it if we don't want it.

Comment (celestehorgan) [reply howto/versioning-documentation.md:8]: Please remove it.

File: howto/versioning-documentation.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+# Technical Documentation Versioning with Hugo & Netlify
+
+Technical Documents Versioning is an intersection of:
+
+**Changes** + **Language** + **Navigation** + **Search**
+
+For small to medium sized sites using one language/location, a `folder based method` is likely the best method to balance these considerations.
```
Comment (celestehorgan) [inline howto/versioning-documentation.md:15]:
```suggestion
For small to medium sized sites using one language/location, a folder based method is likely the best method to balance these considerations.
```

File: howto/versioning-documentation.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+ <a name="footnote4">4</a>: [Pursley, B.](https://github.com/brianpursley), [Horgan, C.](https://github.com/celestehorgan), [Takahashi, S.](https://github.com/shuuj3) (2020, July 21). _kubernetes/website_. GitHub.  Retrieved February 2, 2021 from [https://github.com/kubernetes/website/blob/072d4b41b45f5311538c24d375432a755f9e3f4c/layouts/partials/navbar-version-selector.html](https://github.com/kubernetes/website/blob/072d4b41b45f5311538c24d375432a755f9e3f4c/layouts/partials/navbar-version-selector.html) 
+
+---
+
+This page has been adapted from the [Technical Documentation Versioning slide show](https://technical-documentation-versioning.netlify.app/), its source can be found here: [https://github.com/nate-double-u/technical-documentation-versioning](https://github.com/nate-double-u/technical-documentation-versioning). 
+
+![Creative Commons License](https://i.creativecommons.org/l/by/4.0/88x31.png)  
+This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
```
Comment (celestehorgan) [inline howto/versioning-documentation.md:207]: Remove this too! :)

**Review by celestehorgan — COMMENTED** [review]
The rest of the file LGTM, thank you so much for adding this!

File: howto/versioning-documentation.md (hunk 3 above)
Comment (nate-double-u) [reply howto/versioning-documentation.md:207]: right, suppose it's redundant here...

**Pull request merged**